FinancialTracker/Helpers/ViewControllerFactory.swift

```diff
@@ -14,59 +14,76 @@ struct StoryboardRepresentation {
     let storyboardId: String
 }
 
-enum TypeOfViewController {
-    case navigation(TypeOfNavigationController)
+enum StoryBoardType: String {
+    case main = "Main"
+}
+
+enum ViewControllerType {
+    case navigation(NavigationControllerType)
     case entry
     case register
     case login
     case balance
     case home
     case expense
+    
+    var value: String {
```

Comment:
No need for this computed `value` property. Use a String rawValue for the enum instead.

Reply:
The idea is that case navigation accepts property of type NavigationControllerType so i cannot set the enum value to be String